conf/app.routes

```diff
@@ -51,7 +51,7 @@ POST          /tax-enrolments/users/:userId/enrolments/:enrolmentKey
 POST          /registration/relationship                                                             @uk.gov.hmrc.agentsexternalstubs.controllers.DesStubController.authoriseOrDeAuthoriseRelationship
 GET           /registration/relationship/utr/:utr                                                    @uk.gov.hmrc.agentsexternalstubs.controllers.DesStubController.getLegacyRelationshipsByUtr(utr: String)
 GET           /registration/relationship/nino/:nino                                                  @uk.gov.hmrc.agentsexternalstubs.controllers.DesStubController.getLegacyRelationshipsByNino(nino: String)
-GET           /registration/relationship                                                             @uk.gov.hmrc.agentsexternalstubs.controllers.DesStubController.getRelationship(idtype: Option[String] ?= None, `ref-no`: Option[String] ?= None, arn: Option[String] ?= None, agent: Boolean, `active-only`: Boolean, regime: String, from: Option[String] ?= None, to: Option[String] ?= None, relationship: Option[String] ?= None, `auth-profile`: Option[String] ?= None)
+GET           /registration/relationship                                                             @uk.gov.hmrc.agentsexternalstubs.controllers.DesStubController.getRelationship(idtype: Option[String] ?= None, `ref-no`: Option[String] ?= None, `referenceNumber`: Option[String] ?= None, arn: Option[String] ?= None, agent: Boolean, `active-only`: Boolean, regime: String, from: Option[String] ?= None, to: Option[String] ?= None, relationship: Option[String] ?= None, `auth-profile`: Option[String] ?= None)
```

Comment:
I'm not sure if we still need this other code `ref-no`: Option[String] ?= None, as we now call it referenceNumber?

Reply:
My understanding is that we need to be able to handle both

Reply:
yes both ref-no for DES and referenceNumber for IF. Eventually we will remove ref-no (once IF fully up and running)